File: Data Structures & Algorithms/remove-element/submission-1.cpp
class Solution {
public:
    int removeElement(vector<int>& nums, int val) {
        vector<int> temp;
        for(int el : nums){
            if(el!=val) temp.push_back(el);
        }
        for(int i=0; i<temp.size(); i++){
            nums[i]=temp[i];
        }
        return temp.size();
    }
};

File: Data Structures & Algorithms/rotate-array/submission-1.java
class Solution {
    private void swap(int[] nums, int i, int j){
        int temp = nums[i];
        nums[i] = nums[j];
        nums[j] = temp;
    }
    public void rotate(int[] nums, int k) {
        int n = nums.length;

        while(k>0){
            int ind = n-1;

            while(ind>0){
                swap(nums, ind, ind-1);
                ind--;
            }
            k--;
        }
    }
}